### ISSUE TITLE
`scatterlines` ignores attributes

### ISSUE THREAD

**Issue by devmotion** [JuliaPlots/Makie.jl#854]
It seems recent versions of CairoMakie and WGLMakie (probably, it's a more fundamental issue in AbstractPlotting but I tried only these backends) ignore attributes to `scatterlines`. E.g., the following code
```julia
scatterlines!(1:10, rand(10); color=:red, linecolor=:blue, marker=:x)
```
does not work. It worked some months ago though - did the syntax change?

**Comment by jkrumbiegel** [comment]
Thanks, this one was also not in the docs. The implementation seemed a bit buggy to me, so I cleaned it up a little in https://github.com/JuliaPlots/AbstractPlotting.jl/pull/644

**Comment by devmotion** [comment]
Thank you!

**Comment by devmotion** [comment]
I just updated to the latest release of AbstractPlotting. I noticed that `markercolor`, `color`, `marker`, and `markersize` work as expected but that e.g. `linewidth` is not supported. It seems only the attributes listed explicitly in https://github.com/JuliaPlots/AbstractPlotting.jl/blob/2293e85f986718a03ce8f7022563bfa3e9241435/src/basic_recipes/scatterlines.jl#L29-L40 are supported but not other attributes of `lines` and `scatter`. In general, I guess the main problem with supporting all attributes of `lines` and `scatter` is that they might be conflicting, and therefore special attributes such as `markercolor` instead of `color` are needed?
